docs/self-managed/operational-guides/multi-region/dual-region-ops.md

```diff
@@ -342,7 +339,7 @@ curl -XGET 'http://localhost:9600/actuator/exporters'
   </summary>
 </details>
 
-2. Based on the Exporter API you will send a request to the Zeebe Gateway to disable the Elasticsearch exporter to the lost region.
+2. Based on the [Exporter APIs](../../zeebe-deployment/operations/cluster-scaling.md) you will send a request to the Zeebe Gateway to disable the Elasticsearch exporter to the lost region.
```

Comment:
I thought so first as well but it's not documented there, so removed it for now to not cause confusion.
Afaik not 100% documented yet.